tests/combine_results_test.py

```diff
@@ -260,18 +260,15 @@ def test_main_for_combine_all(self, mock_read_header_json, mock_create_df, mock_
     @patch('bin.combine_results.read_header_json')
     def test_main_for_pbp_typer(self, mock_read_header_json, mock_create_df, mock_write_pandas_output, mock_get_arguments):
         args = mock_get_arguments.return_value.parse_args()
-        args.which = "surface_typer"
+        args.which = "pbp_typer"
         header_dict = self.header_dict
         mock_read_header_json.return_value = header_dict
         mock_create_df.return_value = 'foobar1'
         main()
```

Comment:
I think it helps readability and sets a structure when the setup, action, and assertion parts of a test are visually separated, e.g. by an empty line.

In this case `main()` would have an empty line above and one below itself.

Reply:
Good shout. I admit the unit tests are a bit messy - have added some more line breaks here and in other test scripts